File: java/src/HelloWorld.java



public class HelloWorld {

    static String a_class = "abcdefg";

    public static void main(String []args){
        System.out.println("Hello World");
        System.out.println(a_class.length());
        System.out.println(a_class.substring(2,5));
        System.out.println(a_class.replace("a","A"));
        System.out.println(a_class.startsWith("a"));
    }
}

File: python/HelloWorld.py

import keyword

class HelloWorld(object):

    a_class = "abcdefg"

    def printHellWorld(self):
        print("Hello World")
        print(keyword.kwlist)
        print(self.a_class[2:5])
        print(len(self.a_class))
        print(self.a_class.replace("a","A"))
        print(self.a_class.startswith("a"))

if __name__ == "__main__":
    HelloWorld().printHellWorld()